smk-config.json

```diff
@@ -15,7 +15,8 @@
             ],
             "zoom": 6.75
         },
-        "baseMap": "Streets"
+        "baseMap": "Streets",
+        "esriApiKey": "AAPK869398b518894ae8aabcb897633e0db3LX46wT9TRnePMwjImmKZcToS2KGrpeJN66cCNnMKdjhkCLefNJn9OYKagquWaF6b"
```

Comment:
Are you using a different api key for each map? If not, we should be so we are less likely to bump into the tile limit.

Reply:
Yes, I'm creating an API key in the keymaker account for each hosted SMK app. I'm also coordinating with Caitlin and waiting on her go-ahead before moving any basemap changes to production.